lib/genome/online_updaters/docm/api_client.rb

```diff
@@ -0,0 +1,29 @@
+module Genome; module OnlineUpdaters; module Docm
+  class ApiClient
+    def variants
+      @variants = JSON.parse(get_page(variant_url))
```

Comment:
It looks like the results here are stored in an instance variable (`@variants`) which is never referenced again. Each call to this method will make the HTTP request and just rewrite whatever was in `@variants` before. If you wanted to memoize the result, you'll want to do an `@variants ||=`.  

Though this doesn't change the correctness of the method. It will work fine as written!